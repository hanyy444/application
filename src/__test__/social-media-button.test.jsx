import React from "react";
import { render, screen } from '@testing-library/react';

import SocialMediaButton from "../components/social-media-button/social-media-button.component";

describe('Test social media button', () => {
    expect.assertions(2);
    test('renders without crashing', () => {
        render(<SocialMediaButton />)
        expect(screen.getByTestId('social-media-button')).toBeInTheDocument();
        expect(screen.getByTestId('social-media-button')).toMatchSnapshot();
    })
})