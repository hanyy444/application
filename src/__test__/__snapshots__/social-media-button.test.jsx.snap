// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test social media button renders without crashing 1`] = `
<span
  class="sc-bczRLJ kdRmkh"
  data-testid="social-media-button"
/>
`;
